repositories.bzl

```diff
@@ -234,14 +235,6 @@ filegroup(
             actual = "@mixerapi_git//:tcp_cluster_rewrite_config_cc_proto",
         )
 
-load(":protobuf.bzl", "protobuf_repositories")
```

Comment:
Can you delete that file as well? Thanks!

Reply:
done